app/Models/Multiplayer/PlaylistItem.php

```diff
@@ -21,6 +21,7 @@
  * @property int $owner_id
  * @property int|null $playlist_order
  * @property json|null $required_mods
+ * @property bool $freestyle
```

Comment:
needs to be added into `$casts` as well.

(unrelated to this, why is the `mods` cast of this `object` but `DailyChallengeQueueItem` `array` 🤔 )